tests/integration/ovs_test.py

```diff
@@ -15,106 +14,102 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
-#
```

Comment:
The header “fix” better be consistent with the whole project, therefore I suggest you do this in a separate PR project wide.

Reply:
Hoped to sneak it in... :crying_cat_face: